### PR TITLE
vSphere: clean up registry mirror

### PR DESCRIPTION
**Pull request by jcpowermac**
Since we have implemented caching for all vSphere jobs this registry mirror can no longer be used - not
that it was anyway because the configuration at least for OCP was incorrect.

The OKD changes recently made have been removed.

Added a samples operator config to remove the openliberty imagestream.

### CODE REVIEW

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-release-master-okd-scos-4.17-e2e-vsphere-ovn periodic-ci-openshift-release-master-nightly-4.17-e2e-vsphere-ovn

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-release-master-okd-scos-4.17-e2e-vsphere-ovn periodic-ci-openshift-release-master-nightly-4.17-e2e-vsphere-ovn

**Comment by jcpowermac** [comment]
/pj-rehearse periodic-ci-openshift-release-master-okd-scos-4.17-e2e-vsphere-ovn periodic-ci-openshift-release-master-nightly-4.17-e2e-vsphere-ovn

**Comment by vrutkovs** [comment]
I'd rather not touch anything OKD-related
/unassign

File: ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh [hunk 1]
```diff
@@ -675,6 +675,19 @@ aws|aws-arm64|aws-usgov)
       inject_spot_instance_config "${dir}" "masters"
     fi
     ;;
+vsphere*)
```
Comment (vrutkovs) [inline ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh:678]: If its OKD specific it needs to be a separate step

**Pull request merged**